blog/views.py

```diff
@@ -62,13 +62,9 @@ def article_edit(id):
         abort(404)
     article.title = request.form.get('title')
     article.content = request.form.get('content')
-    errors = []
     if article.title != '' and clean_html(article.content) != '':
         article.save()
-    else:
-        errors.append('Your article must have a title and a content')
-    return render_template('blog/article.html', article=article, edit=True,
-                           errors=errors)
+    return redirect(url_for_article(article))
```

Comment:
@vinyll I have an issue with this redirection, the things is if the user remove the title of the article or the content, we don't handle the errors, I don't know how to give the errors threw this redirection.